test/unit/test-helper.sh

```diff
@@ -64,3 +64,10 @@ function mock_gcloud_function() {
   }
   "$@" 2>&1
 }
+
+# Convenience method to display a test failure and exit the script.
+# Parameters: $1 - message to display.
+function failed() {
```

Comment:
this function name is also used as variable name in `scripts/presubmit-tests.sh`, is it on purpose?

Reply:
No, and one won't affect the other. Anyway, I'll rename for clarity.